extensions/wikia/AdEngine/js/context/adContext.js

```diff
@@ -88,23 +87,18 @@ define('ext.wikia.adEngine.adContext', [
 	}
 
 	function enableAdMixExperiment(context) {
-		var group = abTest.getGroup('AD_MIX') || '';
-
 		context.opts.adMixExperimentEnabled = !!(
-			group.indexOf('AD_MIX_') === 0 &&
 			isPageType('article') &&
 			context.targeting.skin === 'oasis' &&
+			context.targeting.hasFeaturedVideo &&
 			geo.isProperGeo(instantGlobals.wgAdDriverAdMixCountries)
 		);
 
-		context.opts.adMix1Enabled = context.opts.adMixExperimentEnabled && abTest.getGroup('AD_MIX').indexOf('AD_MIX_1') === 0;
-		context.opts.adMix3Enabled = context.opts.adMixExperimentEnabled && abTest.getGroup('AD_MIX').indexOf('AD_MIX_3') === 0;
-
-		context.slots.adMixToUnblock = [];
+		// TODO remove admix1 in ADEN-5443
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)